ui/desktop/app/styles/app.scss

```diff
@@ -123,6 +123,20 @@
     margin: sizing.rems(xs);
   }
 
+  .button-window-close {
+    &:hover {
+    background: var(--failure);
```

Comment:
It looks like this formatting didn't get fixed, were the git hooks bypassed? Can you re-run the formatter for this file

Reply:
It should not get bypassed! Fixing.